gazebo/rendering/LensFlare.cc

```diff
@@ -212,8 +215,10 @@ namespace gazebo
       // create dummy camera for occlusion checking
       // Needed so we can reuse Scene::FirstContact function which expects
       // a gazebo camera object
-      std::vector<Ogre::Camera *> ogreEnvCameras =
-          _wideAngleCam->OgreEnvCameras();
+      std::vector<Ogre::Camera *> ogreEnvCameras = _wideAngleCam->OgreEnvCameras();
+      
+      // set dummy camera properties based on env cam
+      Ogre::Camera *cam = ogreEnvCameras[0];
```

Comment:
I think the `cam` and `ogreEnvCameras` variables can be removed if all instances of `cam` are replaced by `_wideAngleCam`

Reply:
that's a good point